app/views/whatsrunningwhere/WhatsRunningWherePage.scala.html

```diff
@@ -89,7 +90,13 @@ <h1>What's Running Where? @platform.displayName</h1>
 
                 @environments.map { env =>
                     @wrw.versions.find(_.environment == env).map { version =>
-                        <td id="row@{i}_version_@{env.asString}">@version.versionNumber.asString</td>
+                        <td id="row@{i}_version_@{env.asString}"
+                        @if(showDiff && env == Environment.Production) {
+                            @if(!matchesStaging(wrw, version)){
```

Comment:
I don't think we should add this class when prod is ahead of staging or if staging is missing. At the moment `matchesStaging ` function is doing equality check.

Reply:
Or is that the intention to show a visual difference if there is any version difference.

Reply:
Yes so the acceptance criteria for this were to highlight if(stagingVersion!=productionVersion), I will however get feedback on making that if(stagingVersion>productionVersion)

The case of staging being missing is something I haven't considered however, will look into that.